.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I don't think this file should be a part of this PR

Reply:
I also didn't want to add this file but during the VS commit it automatically got committed. I will remove it or anyone can.